rules/ask-open-ended-questions/rule.md

```diff
@@ -14,45 +14,34 @@ redirects:
 
 ---
 
-The goal of any outbound call is to get the person on the other end of the line involved. The way to do this is to employ a knockout combo. If you were a boxer, you'd follow up a jab with a cross hook and an uppercut! (the good ole one-two-three knockout!). If you're a telemarketer you follow the YES ladder up with open-ended questions. Here's how most of your combos should sound:
+The goal of any outbound call is to get the person on the other end of the line involved. The way to do this is to employ a knockout combo. If you were a boxer, you'd follow up a jab with a cross hook and an uppercut! (the good old one-two-three knockout!). If you're a telemarketer you follow the ["Yes Ladder](/build-a-yes-ladder-to-your-outbound-script) up with open-ended questions.
```

Comment:
I like good 'ole' it adds personality to the page

Also looks like an unclosed/accidental quote here: ["Yes Ladder]

Reply:
Agreed and fixed